### PR TITLE
Feat/assets service

### PR DESCRIPTION
**Pull request by thomaslennan**
New VantageAssetsService for retrieving misc assets such as help links, etc.

See usage in demo component.

### CODE REVIEW

File: src/docs-ui/app/demos/demos.component.ts [hunk 1]
```diff
@@ -62,4 +66,11 @@ export class DemosComponent implements OnInit, OnDestroy {
       error,
     });
   }
+
+  getHelpJSON(): void {
+    this.assetsService.getHelpJSON().subscribe((helpJSON: IHelpAssets) => {
+      console.log('HERE', helpJSON);
```
Comment (owilliams320) [inline src/docs-ui/app/demos/demos.component.ts:72]: can you remove this console log

Comment (thomaslennan) [reply src/docs-ui/app/demos/demos.component.ts:72]: Ooops.

File: src/lib/assets/package.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "ngPackage": {
+    "lib": {
+      "entryFile": "index.ts",
+      "umdModuleIds": {
+        "@covalent/http": "covalent.http",
+        "@ngx-translate/core": "ngx-translate.core"
```
Comment (owilliams320) [inline src/lib/assets/package.json:7]: Looks like ngx translate is not be used here. Can we remove this?

Comment (thomaslennan) [reply src/lib/assets/package.json]: Done.

**Pull request merged**